From 34870178989b2c75f42946ee4e2010ad883bab34 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Mon, 30 May 2022 19:42:08 +0800
Subject: [PATCH] Revert "Linux 5.10.119"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 56c31ac1d8aadb706ea977c097c714762b3fcfdd.

Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b442cc5bbfc3..f9210e43121d 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 119
+SUBLEVEL = 118
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.1

